VideoLocker/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java

```diff
@@ -52,6 +45,18 @@ public CoursesAsyncLoader(Context context) {
         return result;
     }
 
+    private List<EnrolledCoursesResponse> getEnrolledCourses() {
+        List<EnrolledCoursesResponse> ret = null;
+
+        try {
+            ret = environment.getServiceManager().getEnrolledCourses(false);
+        } catch (Exception e) {
```

Comment:
This generic try-catch is improperly handling an 401 oauth exception. 

We want to use https://github.com/edx/edx-app-android/blob/master/VideoLocker/src/main/java/org/edx/mobile/user/UserAPI.java#L71

instead of the old non retrofit api.